MarvelApiClient/src/main/java/com/karumi/MarvelApiClient/MarvelApiException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ *   Copyright (C) 2015 Karumi.
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.karumi.marvelapiclient;
+
+public class MarvelApiException extends RuntimeException {
```

Comment:
Use a ``RuntimeException`` here is not a good idea. I'd like to be more explicit with the usage of exceptions to indicate http error codes. I'd change this class signature to extend ``Exception`` instead. If we don't do this and the library user forget to add the catch, we are going to crash his app :( Change to ``Expection`` instead of ``RuntimeException`` will fix this problem :)